From 45d08dcdf4c4baf13864db25d848d55748ed0ee2 Mon Sep 17 00:00:00 2001
From: "Dr. Git" <drgitx@gmail.com>
Date: Fri, 16 Apr 2021 23:17:29 +0000
Subject: [PATCH 30/35] arm64: dts: qcom: sdm845-oneplus-common: remove panel
 reset gpio

Clean up DT now that the panel driver no longer touches the reset GPIO.
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index 8f617f7b6d34..8f3f5c687b4a 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -314,8 +314,6 @@ display_panel: panel@0 {
 
 		vddio-supply = <&vreg_l14a_1p88>;
 
-		reset-gpios = <&tlmm 6 GPIO_ACTIVE_LOW>;
-
 		pinctrl-names = "default";
 		pinctrl-0 = <&panel_reset_pins &panel_te_pin &panel_esd_pin>;
 
-- 
2.30.2

